client/document/index.jsx

```diff
@@ -191,6 +191,7 @@ class Document extends Component {
 							src={ getBilmurUrl() }
 							data-provider="wordpress.com"
 							data-service="calypso"
+							data-customproperties={ `{"route": "${ sectionName }"}` }
```

Comment:
Confirmed with `bilmur`'s code that `data-customproperties` expects data in a stringified JSON 👍